src/database/seeders/20190819153902-user.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         lastName: 'Williams',
         email: 'jammy@gmail.com',
         phoneNo: '2347032123304',
-        password: 'jammy11',
+        password: 'jammy11167',
```

Comment:
why change the password?

Reply:
It was failing due to the password length